src/test/java/duke/command/ParserTest.java

```diff
@@ -0,0 +1,27 @@
+package duke.command;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import org.junit.jupiter.api.Test;
+
+import duke.exception.InvalidCommandException;
+
+public class ParserTest {
```

Comment:
Maybe some descriptions for test methods can be provided. However, I think the JavaDoc method descriptions for the rest of your classes are clear and concise.